leveldown.js

```diff
@@ -229,7 +229,7 @@ Multilevel.prototype._close = function (cb) {
 
   this._clearRequests(true)
   if (this._streaming) {
-    this._streaming.once('close', cb)
+    this._streaming.once('close', () => cb())
```

Comment:
Can you add a comment here to explain the wrapped callback? Folks won't expect `close` to have an argument (thinking that it's a duplexify stream).